src/lib/apiTypesV2/Entity.cpp

```diff
@@ -55,25 +55,39 @@ Entity::~Entity()
 /* ****************************************************************************
 *
 * Entity::render - 
+*
+* The rendering of JSON in APIv2 depends on the URI param 'options'
+* Rendering methods:
+*   o 'normalized' (default)
+*   o 'keyValues'  (less verbose, only name and values shown for attributes - no type, no metadatas)
+*   o 'values'     (only the values of the attributes are printed, in a vector)
 */
 std::string Entity::render(ConnectionInfo* ciP, RequestType requestType, bool comma)
 {
+  std::string renderMode = "normalized";
+
+  if (ciP->uriParamOptions["keyValues"] == true)
+  {
+    renderMode = "keyValues";
```

Comment:
Unfortunatelly, "keyValues" seems that will not be used at the end. However, the issue is still under discussion, so probably the best strategy is to progress this PR without changes, then fix it in a next PR once it gets clear. What do you think?